pkg/zson/raw.go

```diff
@@ -23,20 +23,20 @@ func EncodeZvals(d *Descriptor, vals []zval.Encoding) (zval.Encoding, error) {
 	return raw, nil
 }
 
-func NewRawAndTsFromZeekTSV(builder *zval.Builder, d *Descriptor, path []byte, data []byte) (zval.Encoding, nano.Ts, error) {
+func NewRawAndTsFromZeekTSV(builder *zval.Builder, d *Descriptor, path []byte, data []byte) (zval.Encoding, zeek.Value, error) {
```

Comment:
I think this will add a heap allocation to every call.

Reply:
Should we pass in a *zeek.Value like we did for the builder?

Reply:
I discussed this with @nwt: we  could pass in a *zeek.Time, or return a nano.Ts and bool like we did previously. 

Since that's going around the zeek.Value API though, it seems like it's something to avoid unless there's a significant performance impact.

So I made some measurements and on a large file (wrccdc conn log) there is no impact. Which leads me to think we should leave this as-is.

Reply:
Perhaps this is a naive question, but why include the ts in the return value? Why not grab the ts from the returned encoding (could use TypeEncoding) in whatever is calling this function? Would this result in a large perf regression?

Reply:
Yes, I think it's for perf reasons. I agree it would be nicer to do what you suggest and I believe there's some desire to rework this API at some point, but ideally not in this bug fix...